From 5b8f1294844e3639c6be802dda81fa853511bf30 Mon Sep 17 00:00:00 2001
From: Murray Cumming <murrayc@murrayc.com>
Date: Fri, 14 Aug 2015 13:50:40 +0200
Subject: configure.ac: Use C++11 if the compiler supports it.

For instance, use --std=c++11 when using g++.
This will let gtkmm-2.24 build against newer glibmm versions.
Applications will need to use --std=c++11 too if glibmm is a newer
version.
Of course, you should be using gtkmm 3 anyway.
---
 configure.ac | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index eef1169..6a72d74 100644
--- a/configure.ac
+++ b/configure.ac
@@ -31,7 +31,7 @@ AM_MAINTAINER_MODE
 AC_ARG_VAR([ACLOCAL_FLAGS], [aclocal flags, e.g. -I <macro dir>])
 
 
-MM_PREREQ([0.9.5])
+MM_PREREQ([0.9.8])
 MM_INIT_MODULE([gdkmm-2.4])
 MM_INIT_MODULE([gtkmm-2.4])
 
@@ -44,6 +44,8 @@ MM_CONFIG_DOCTOOL_DIR([docs])
 AC_SUBST([LIBGTKMM_SO_VERSION], [2:0:1])
 
 AC_PROG_CXX
+MM_AX_CXX_COMPILE_STDCXX_11([noext],[optional])
+
 AC_DISABLE_STATIC
 AC_LIBTOOL_WIN32_DLL
 AC_PROG_LIBTOOL
-- 
cgit v0.11.2

